silverstripe/framework/CVE-2021-25817.yaml

```diff
@@ -0,0 +1,8 @@
+title:     'CVE-2021-25817 XXE: Vulnerability in CSSContentParser'
+link:      https://www.silverstripe.org/download/security-releases/cve-2021-25817
+cve:       CVE-2021-25817
+branches:
+    4.0.x:
+        time:     2021-06-07 22:31:00
+        versions: ['>=4.0.0', '<4.8.0']
```

Comment:
Based on the link the fix is already in 4.7.4?

Reply:
```suggestion
        versions: ['>=4.0.0', '<4.7.4']
```

Reply:
Apologies, yes you're correct.  I have updated this.